tagging/models.py

```diff
@@ -1,22 +1,21 @@
 """
 Models and managers for generic tagging.
 """
-# Python 2.3 compatibility
-try:
-    set
-except NameError:
-    from sets import Set as set
```

Comment:
`set` wasnt used in this file at all?

Reply:
or what was this for?

Reply:
Old python didn't have `set` in the standard lib. We are not using old python any longer :)